kafka/channel/config/200-serviceaccount.yaml

```diff
@@ -20,15 +20,6 @@ metadata:
   labels:
```

Comment:
perhaps we make a split of the remaining two, and move them to 

* 200-controller-serviceaccount.yaml
* 200-webhook-serviceaccount.yaml